### PR TITLE
fix(legacy): allow updating track types code

### PR DESCRIPTION
**Pull request by caveman99**
This is a minor nag i had. Since moving to an indexed track_type model for 3.0 the constraint of the shortlabel being immutable is no longer valid, since it's not used as index any more. This commit removes the readonly flag from that form field.

### CODE REVIEW

File: legacy/public/js/airtime/tracktype/tracktype.js [hunk 1]
```diff
@@ -156,6 +150,7 @@ function initTracktypeData() {
     description: "",
     visibility: "1",
     id: "",
+    analyze_cue_points: true,
```
Comment (jooola) [inline legacy/public/js/airtime/tracktype/tracktype.js:153]: This seem unrelated ?

Comment (caveman99) [reply legacy/public/js/airtime/tracktype/tracktype.js:153]: Err yes. This is part of a bigger thing i am working on. However this field is not initialized in the code as-is. Default for this is 'true' in the schema, so i added it.

Comment (jooola) [reply legacy/public/js/airtime/tracktype/tracktype.js:153]: Please keep this is the "bigger thing" you are working on.

Comment (caveman99) [reply legacy/public/js/airtime/tracktype/tracktype.js]: Removed, sorry.

**Review by jooola — APPROVED** [review]
